landingpage/index.html

```diff
@@ -0,0 +1,152 @@
+
+    <div class="row featurette">
+        <div class="col-md-7">
+            <h2 class="featurette-heading">Student <span class="text-muted"></span></h2>
+            <p class="lead">As a student, you will be able to search for high schools using various filters such as
+                location, safety, programs offered, school performance and much more. By filling out a simple
+                application form, you will be able to apply to as many schools as you wish to with just one click of a
+                button.</p>
+        </div>
+        <div class="col-md-5">
+            <img class="featurette-image img-fluid mx-auto" src="{% static '/landingpage/students.jpeg' %}"
+                 alt="Generic placeholder image">
+        </div>
+    </div>
+
+    <hr class="featurette-divider">
+
+    <div class="row featurette">
+        <div class="col-md-7">
+            <h2 class="featurette-heading">Admission Staff<span class="text-muted"></span></h2>
+            <p class="lead">As an admissions staff of a high school, OneApply provides you with myriad of features to
+                view, sort and manage millions of applications. Sit back and relax while OneApply does everything for
+                you from managing applications to updating the number of seats available to enroll students.</p>
+        </div>
+        <div class="col-md-5">
+            <img class="featurette-image img-fluid mx-auto" src="{% static '/landingpage/teacher.jpeg' %}"
+                 alt="Generic placeholder image">
+        </div>
+    </div>
+
+    <!--<hr class="featurette-divider">-->
+
+    <!-- /END THE FEATURETTES -->
+
+
+    <!-- FOOTER -->
+    <footer>
+        <p class="float-right"><a href="#">Back to top</a></p>
+        <p>&copy; 2017 Company, Inc. &middot; <a href="#">Privacy</a> &middot; <a href="#">Terms</a></p>
+    </footer>
+
+</div><!-- /.container -->
+
+
+<!-- Bootstrap core JavaScript
+================================================== -->
+<!-- Placed at the end of the document so the pages load faster -->
+<script src="https://code.jquery.com/jquery-3.1.1.slim.min.js"
+        integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n"
+        crossorigin="anonymous"></script>
+<script>window.jQuery || document.write('<script src="../../assets/js/vendor/jquery.min.js"><\/script>')</script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js"
+        integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb"
+        crossorigin="anonymous"></script>
+<script src="{% static '/js/bootstrap.min.js'%}"></script>
+<!-- Just to make our placeholder images work. Don't actually copy the next line! -->
+<!--script src="../../assets/js/vendor/holder.min.js"></script>
+<!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
+<!--script src="../../assets/js/ie10-viewport-bug-workaround.js"></script>
+</body>
+</html>
```

Comment:
Even though it might not be necessary for HTML, it might be a good idea to have a new line at the end of this.
Also, how does this file compare to the templates/landingpage/index.html because it looks very similar to me.

Reply:
solved, and delete extra file